src/sentry/static/sentry/app/components/group/sidebar.jsx

```diff
@@ -69,6 +69,28 @@ const GroupSidebar = React.createClass({
     }
   },
 
+  getNotificationText() {
+    let group = this.getGroup();
+    let reasons = new Map([
```

Comment:
Just use a `{}` because it's unclear to me that this works in IE11 according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Browser_compatibility

Also, might as well move up as a `const` to module level.